### PR TITLE
master-lps-123651

### PR DESCRIPTION
**Pull request by Jeremy-Ch**
https://issues.liferay.com/browse/LPS-128424

### CODE REVIEW

**Comment by Jeremy-Ch** [comment]
ci:test:sf

**Comment by austinchiang** [comment]
**Reviewing...** :hourglass_flowing_sand: :sunglasses: :headphones:

File: modules/apps/document-library-opener/document-library-opener-google-drive-test/src/testFunctional/GoogleDrive.testcase [hunk 1]
```diff
@@ -424,6 +424,20 @@ definition {
 
 		Navigator.gotoPage(pageName = "Documents and Media Page");
 
+		DMNavigator.gotoEditDocumentPG(dmDocumentTitle = "Untitled");
+
+		AssertTextEquals(
```
Comment (austinchiang) [inline modules/apps/document-library-opener/document-library-opener-google-drive-test/src/testFunctional/GoogleDrive.testcase:557]: @Jeremy-Ch there are a lot of redundant assertions here. I would rather there be a macro for this that can cover multiple cases so we don't need to be using functions at the test case level. Can you file a test improvement ticket for this?

**Comment by austinchiang** [comment]
**Pull request submitted:** https://github.com/liferay-lima/liferay-portal/pull/1590<br></br>

**Pull request closed without merging**